.ocamlformat

```diff
@@ -1,4 +1,4 @@
-version=0.15.1
+version=0.16.0
```

Comment:
hold off on this change for now. I prefer to update ocamlformat in vscode, dune, lsp all at once

Reply:
Indeed, I will revert that commit.